scss/components/_search-bar.scss

```diff
@@ -26,27 +26,27 @@
   border-radius: 4px 0 0 4px;
   margin-right: -2px;
   height: 3.6rem;
-  width: 12rem;
+  width: 14rem;
   float: left;
 }
 
-.combo__select {
-  width: 12rem;
-}
-
-.combo__input {
-  width: calc(100% - 12rem)
+.combo--search {
+  .combo__input {
+    border-radius: 0 4px 4px 0;
+    width: calc(100% - 14rem)
+  }
 }
 
 // BREAKPOINT: MEDIUM
 @include media($med) {
   .combo--search {
     .combo__input {
-      width: calc(100% - 16.6rem);
+      border-radius: 0 0 0 0;
```

Comment:
Shorthand form for property `border-radius` should be written more concisely as `0` instead of `0 0 0 0`